test/e2e_node/runner/remote/run_remote.go

```diff
@@ -244,6 +244,10 @@ func main() {
 					klog.Fatalf("Could not retrieve list of images based on image prefix %q and family %q: %v",
 						imageConfig.ImageRegex, imageConfig.ImageFamily, err)
 				}
+				if len(images) == 0 { // if we have no images we can't run anything
```

Comment:
It will be cleaner to put this into the function rather than outside. See #91543